### ISSUE TITLE
pgcli crashes when PAGER is an empty string.

### ISSUE THREAD

**Issue by jonathanslenders** [dbcli/pgcli#327]
``` bash
jonathan @ arch-jonathan [master] ~/git/pgcli
[pt] > PAGER= pgcli --user postgres
```

``` python
Traceback (most recent call last):
  File "/home/jonathan/env/pt/bin/pgcli", line 9, in <module>
    load_entry_point('pgcli==0.19.0', 'console_scripts', 'pgcli')()
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/home/jonathan/git/pgcli/pgcli/main.py", line 461, in cli
    pgcli.run_cli()
  File "/home/jonathan/git/pgcli/pgcli/main.py", line 353, in run_cli
    click.echo_via_pager('\n'.join(output))
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/termui.py", line 203, in echo_via_pager
    return pager(text + '\n', color)
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/_termui_impl.py", line 263, in pager
    return _pipepager(text, os.environ['PAGER'], color)
  File "/home/jonathan/env/pt/lib/python3.4/site-packages/click-4.1-py3.4.egg/click/_termui_impl.py", line 292, in _pipepager
    if color is None and cmd_detail[0] == 'less':
IndexError: list index out of range
```

It's a bug in click. I created an issue there: https://github.com/mitsuhiko/click/issues/403
Just make sure to upgrade when that's fixed.

**Comment by amjith** [comment]
Looks like it's fixed in click. But they don't release often. So I'll keep an eye on it and bump the minimum version when it is released. Thanks!

**Comment by j-bennet** [comment]
This was released in 5.0: https://github.com/pallets/click/commit/349875d795837a8ca1173621001bb585e1640207.
